google_cloud_http_target_task.js

```diff
@@ -0,0 +1,63 @@
+const {CloudTasksClient} = require('@google-cloud/tasks');
+const yenv = require('yenv')
+const { v4: uuidv4 } = require('uuid');
+const fs = require('fs')
+const os = require('os');
```

Comment:
is this used anywhere?

Reply:
No, I have already fixed it in the new commit.